.travis/yapf.sh

```diff
@@ -9,15 +9,24 @@ builtin cd "$(dirname "${BASH_SOURCE:-$0}")"
 ROOT="$(git rev-parse --show-toplevel)"
 builtin cd "$ROOT"
 
-lint() {
+format() {
     yapf \
         --style "$ROOT/.style.yapf" \
         --in-place --recursive --parallel \
         -- \
         "$@"
 }
 
-lint_all() {
+format_changed() {
+    # Formats python files that changed in last commit
+    git diff --name-only HEAD~1 HEAD | grep '\.py$' | xargs -P 5 \
+        yapf \
+        --style "$ROOT/.style.yapf" \
+        --in-place --recursive --parallel
```

Comment:
missing --exclude

Reply:
Oops, didn't push the actual latest version, that's changed. I'd really rather avoid adding an --excludes flag. If someone wants to lint specific files/dirs, they can run `yapf.sh --files <files or dirs>`.